src/server.rs

```diff
@@ -0,0 +1,384 @@
+#![allow(unused)]
```

Comment:
could you explain what's debug benefits of having internals of `tokio-reflection` here?

Reply:
debug prints! It is quite helpful while working on tc and we want some debug prints